ml_tools/eolearn/tests/test_sampling.py

```diff
@@ -134,11 +133,8 @@ def test_get_mask_of_samples(small_image: np.ndarray, n_samples: Dict[int, int])
 
 @pytest.fixture(name="eopatch")
 def eopatch_fixture(small_image: np.ndarray) -> EOPatch:
-    t, h, w, d = 10, *small_image.shape, 5
-    eopatch = EOPatch(bbox=BBox((0, 0, 1, 1), CRS(3857)))
-    eopatch.data["bands"] = np.arange(t * h * w * d).reshape(t, h, w, d)
-    eopatch.mask_timeless["raster"] = small_image.reshape(small_image.shape + (1,))
```

Comment:
this is lost

Reply:
test dose not assert what output should be, but just asserts if mask and resolute are aligned